### PR TITLE
added remove friend from trip feature

### LINKED ISSUE #185: Remove friend from trip
### Steps to reproduce
- Go to navigation drawer. Select `My Trips`
- Go to any trip
- There is a list of friends
- There is no way to remove a friend from the trip

### Expected behaviour
- Add an option on the list item to remove the friend from trip

### Endpoint to use
Everything is mentioned in https://github.com/project-travel-mate/server/issues/16

### PR DESCRIPTION
**Pull request by aayushinigam**
# Description
Fixes #185 
screenshot_20180713-231427
screenshot_20180713-231431

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Swati4star** [comment]
@aayushinigam Add a screenshot. Fix the merge conflicts

File: Android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -81,6 +81,9 @@
     <string name="friends_show_title">Friends in this Trip</string>
     <string name="trip_name_updated">Trip name has been updated</string>
     <string name="cannot_edit">Trip name can\'t remain empty!</string>
+    <string name="remove_friend">Remove Friend</string>
+    <string name="remove_friend_message">Are you sure you want to friend?</string>
```
Comment (Swati4star) [inline Android/app/src/main/res/values/strings.xml]: *to remove the

File: Android/app/src/main/java/io/github/project_travel_mate/mytrips/FriendsProfileActivity.java [hunk 2]
```diff
@@ -92,15 +101,25 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
...
         mToken = mSharedPreferences.getString(USER_TOKEN, null);
-        getFriendDetails(String.valueOf(friendId));
+        getFriendDetails(String.valueOf(mFriendId));
         Objects.requireNonNull(getSupportActionBar()).setHomeButtonEnabled(true);
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
     }
 
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.profile_menu, menu);
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/mytrips/FriendsProfileActivity.java]: You can create a different menu file for this. `friend_profile_menu`

File: Android/app/src/main/java/io/github/project_travel_mate/mytrips/FriendsProfileActivity.java [hunk 3]
```diff
@@ -200,9 +219,60 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 // return back to trip activity
                 finish();
                 return true;
+            case R.id.action_sign_out:
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/mytrips/FriendsProfileActivity.java]: The name of the id will also be more relevant, if you create a new menu file

Comment (aayushinigam) [reply Android/app/src/main/java/io/github/project_travel_mate/mytrips/FriendsProfileActivity.java]: Actually there was use of just one item so I used the existing one. But alright I'll do that.

Comment (Swati4star) [reply Android/app/src/main/java/io/github/project_travel_mate/mytrips/FriendsProfileActivity.java]: It's just that we might add new menu items to profile page. Then, that will cause issue

**Comment by Swati4star** [comment]
@aayushinigam Make it show always. add a white cross icon to it. 
Also, you haven't added the friend_profile_menu.xml file to commit.

**Pull request merged**